### ISSUE TITLE
Tempalte: Inaccessible buttons at 200% zoom

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#83]
When the page is zoomed in to 200%, the buttons across the top of the page are removed and are not accessible to users 

"A-Z
Contact
Directories
Locus
Support LUC"

**Comment by syarwick23** [comment]
Issue fixed with new template
